specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/realm.auth.cred.required.correct.user/request.rpt

```diff
@@ -0,0 +1,8 @@
+connect http://localhost:8000/echo/

```

Comment:
NIT: I would suggest using a more generic path here, such as /resource instead of echo.